### ISSUE TITLE
Mistype in writer.js

### ISSUE THREAD

**Issue by AlexAtNet** [joyent/node-asn1#7]
It seems that here is a mistype:

```
  } else {
    throw new InvalidAsn1ERror('Length too long (> 4 bytes)');
  }
```

`InvalidAsn1ERror` -> `InvalidAsn1Error`

**Comment by StefH** [comment]
Solved in https://github.com/mcavage/node-asn1/pull/21

**Comment by stephenwvickers** [comment]
This issue will be addressed in my fork by the end of the week:

https://github.com/stephenwvickers/node-asn1-ber/issues/7

**Comment by stephenwvickers** [comment]
I have seen there is little progress on this project over the past few years.  My project, the [net-snmp][net-snmp] project, depends on this project, so I have cloned it and migrated (i.e. copied) all the outstanding issues and pull requests.

I will be releasing a new version of my version of this project under the asn1-ber namespace on npmjs.org by the end of the week which should address this particular item (I've already pushed the current code to this namespace to reserve it).

I am now tracking all changes under my own copy at [asn1-ber][asn1-ber].  Note that this is not a "fork" but a clone and fresh commit to a new repository.  I have of course retained all copyrights and attributed back the original author.

There are quite a few projects dependant on this project, and I want to make sure there is momentum on the issues and feature requests reported.

[asn1-ber]: https://github.com/stephenwvickers/node-asn1-ber
[net-snmp]: https://github.com/stephenwvickers/node-net-snmp

**Comment by stephenwvickers** [comment]
This issue has been addressed in my version of this project which has now been published to npm:

https://www.npmjs.com/package/asn1-ber

**Comment by melloc** [comment]
Fixed in #33.

**Comment by stephenwvickers** [comment]
Hi @melloc 

I am the current maintainer of the [asn1-ber][1] package which is a clone of this repository.

I cloned this repository due to the lack of activity because my other [net-snmp][2] package depended on it.

I believe I have fixed all the issues outstanding at the time I cloned the repository, and I also authored a lot more documentation in the README.md file.

Now that it has been transferred to Joyent and you are carrying out work on it can I ask what the plans are?

Thanks

Stev

[1]: https://www.npmjs.com/package/asn1-ber
[2]: https://www.npmjs.com/package/net-snmp
